### ISSUE TITLE
"make install" without targetdir fails on install-exec-hook mkdir //usr in cygwin32 environment

### ISSUE THREAD

**Issue by FlyinFistOfJuda** [libressl-portable/portable#234]
I'm trying to do a make install on cygwin32 environment from github 'portable' repository. Source configure runs ok without extra options, make check runs ok, but make install fails if triggered without DESTDIR (but works with ok with DESTDIR). Apparently install-exec-hook tries to create a directory with doublebackslash //usr, which fails:

```
     No need to remake target '../../configure'.
    Finished prerequisites of target file '../../config.status'.
    Prerequisite '../../configure' is older than target '../../config.status'.
   No need to remake target '../../config.status'.
  Finished prerequisites of target file 'Makefile'.
  Prerequisite 'Makefile.in' is older than target 'Makefile'.
  Prerequisite '../../config.status' is older than target 'Makefile'.
 No need to remake target 'Makefile'.
Updating goal targets....
Considering target file 'install-exec-hook'.
 File 'install-exec-hook' does not exist.
 Finished prerequisites of target file 'install-exec-hook'.
Must remake target 'install-exec-hook'.
make[4]: Entering directory '/home/username/projects/libressl/apps/openssl'
Putting child 0x800b81a8 (install-exec-hook) PID 4952 on the chain.
Live child 0x800b81a8 (install-exec-hook) PID 4952
mkdir: cannot create directory ‘//usr’: Read-only file system
/usr/bin/install: cannot create regular file '//usr/local/etc/ssl/cert.pem': No such file or directory
/usr/bin/install: cannot create regular file '//usr/local/etc/ssl/openssl.cnf': No such file or directory
/usr/bin/install: cannot create regular file '//usr/local/etc/ssl/x509v3.cnf': No such file or directory
Reaping losing child 0x800b81a8 PID 4952
make[4]: *** [Makefile:832: install-exec-hook] Error 1
Removing child 0x800b81a8 PID 4952 from chain.
make[4]: Leaving directory '/home/username/projects/libressl/apps/openssl'
Reaping losing child 0x8015af78 PID 10256
make[3]: *** [Makefile:769: install-exec-am] Error 2
Removing child 0x8015af78 PID 10256 from chain.
make[3]: Leaving directory '/home/username/projects/libressl/apps/openssl'
Reaping losing child 0x801521e8 PID 51716
make[2]: *** [Makefile:713: install-am] Error 2
Removing child 0x801521e8 PID 51716 from chain.
make[2]: Leaving directory '/home/username/projects/libressl/apps/openssl'
Reaping losing child 0x80056960 PID 51728
make[1]: *** [Makefile:353: install-recursive] Error 1
Removing child 0x80056960 PID 51728 from chain.
make[1]: Leaving directory '/home/username/projects/libressl/apps'
Reaping losing child 0x80056a30 PID 14564
make: *** [Makefile:443: install-recursive] Error 1
Removing child 0x80056a30 PID 14564 from chain.
```

**Comment by kinichiro** [comment]
This issue occurs on Cygwin64, too.
I think unneeded slash after $(DESTDIR) should be removed in apps/openssl/Makefile.am.
PR https://github.com/libressl-portable/portable/pull/236 will fix this issue.

**Comment by busterb** [comment]
This is fixed with #236
